app/src/main/java/com/keylesspalace/tusky/components/viewthread/ViewThreadViewModel.kt

```diff
@@ -86,10 +83,7 @@ class ViewThreadViewModel @Inject constructor(
             eventHub.events
                 .collect { event ->
                     when (event) {
-                        is FavoriteEvent -> handleFavEvent(event)
-                        is ReblogEvent -> handleReblogEvent(event)
-                        is BookmarkEvent -> handleBookmarkEvent(event)
-                        is PinEvent -> handlePinEvent(event)
+                        is StatusChangedEvent -> handleStatusChangedEvent(event.status)
```

Comment:
Should `StatusEditedEvent` be removed here then?

Reply:
ok yes that is also redundant now